docker/docker-compose.yml

```diff
@@ -0,0 +1,46 @@
+version: '3.8'
+
+services:
+  build:
+    working_dir: /cumulus-dashboard
+    image: dashboard-build:${TAG-}
+    build:
+      dockerfile: docker/Dockerfile
+      context: ../
+      target: build
```

Comment:
The stuff below is duplicated because we want to allow a user to build a deployable dashboard bundle from docker as well as create a docker image that can be run to test the dashboard.